backend/dataall/modules/shares_base/services/share_notification_service.py

```diff
@@ -109,6 +110,38 @@ def notify_persistent_email_reminder(self, email_id: str):
         )
         return notifications
 
+    def notify_managed_policy_limit_exceeded_action(self, email_id: str):
+        share_link_text = ''
+        if os.environ.get('frontend_domain_url'):
+            share_link_text = (
+                f'<br><br>Please visit data.all <a href="{os.environ.get("frontend_domain_url")}'
+                f'/console/shares/{self.share.shareUri}">share link</a> '
+                f'to view more details.'
+            )
+
+        msg_intro = f"""Dear User,
+        
+        We are contacting you because for a share requested by {email_id} failed because no new managed policy can be attached to your IAM role {self.share.principalRoleName}.
+        Please check the service quota for the managed policies that can be attached to a role in your aws account and increase the limit.
```

Comment:
nit: for the IAM Managed Policies that can be attached to an IAM role in your AWS account

Reply:
Yes. I added this sentence. Thanks @dlpzx